inc/class-wp-forms-api.php

```diff
@@ -910,11 +916,31 @@ static function process_element( $element, &$values, &$input ) {
 			else if( isset( $element['#type'] ) && $element['#type'] != 'composite' ) {
 				$element['#value'] = sanitize_text_field( $element['#value'] );
 			}
+
+			// Validate element value and place result in #valid
+			if( !empty( $element['#validator'] ) ) {
+				$validators = (array) $element['#validator'];
```

Comment:
So `[ 'class or object', 'method' ]` is a valid callable, and since it is already an array, `(array)` will not change it and thus the iteration will be incorrect.